server/app/mutations/grid_services/common.rb

```diff
@@ -352,6 +352,7 @@ def common_validations
               integer :initial_delay, default: 10
             end
           end
+          string :stop_signal, matches: /^((SIG)([A-Z0-9]+|RTMIN\+\d+|RTMAX-\d+)|\d+)$/i
```

Comment:
Fix this to `/\A...\z/`

Reply:
Fixed, and also learned something new today -- didn't know there was a difference.